docs/codecs/blosc/v1.0.rst

```diff
@@ -0,0 +1,141 @@
+
+
+Status of this document
+=======================
+
+This document is a **Work in Progress**. It may be updated, replaced
+or obsoleted by other documents at any time. It is inappapropriate to
+cite this document as other than work in progress.
+
+Comments, questions or contributions to this document are very
+welcome. Comments and questions should be raised via `GitHub issues
+<https://github.com/zarr-developers/zarr-specs/labels/codecs-blosc-v1.0>`_. When
+raising an issue, please add the label "codecs-blosc-v1.0".
+
+This document was produced by the `Zarr core development team
+<https://github.com/orgs/zarr-developers/teams/core-devs>`_.
+
+
+Document conventions
+====================
+
+Conformance requirements are expressed with a combination of
+descriptive assertions and [RFC2119]_ terminology. The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in the normative
+parts of this document are to be interpreted as described in
+[RFC2119]_. However, for readability, these words do not appear in all
+uppercase letters in this specification.
+
+All of the text of this specification is normative except sections
+explicitly marked as non-normative, examples, and notes. Examples in
+this specification are introduced with the words "for example".
+
+
+Chunk encoding/decoding with Blosc
+==================================
+
+@@TODO define how chunks are encoded and decoded
+@@TODO be sure to clarify that the encoded data should conform to the Blosc file format
+
+Chunks are encoded and decoded using the compression algorithm implemented in the
+`C-Blosc library version 1 <https://github.com/Blosc/c-blosc>`_ or
+`C-Blosc library version 2 <https://github.com/Blosc/c-blosc2>`_,
+and encoded data should be stored as it is to file.
+
+The compressor can be configured through the following parameters:
+
+- The compression level is an integer from 0 to 9 which controls the speed and
+  level of compression. A level of 1 is the fastest compression method and
+  produces the least compression, while 9 is slowest and produces the most
+  compression. Compression is turned off completely when level is 0.
+- The shuffling method is an integer from -1 to 2 which controls the way bytes or
+  bits are rearranged, which can lead to a greater compression.
+  A value of 1 performs byte-wise shuffling, and a value of 2 performs bit-wise
+  shuffling. If a value of -1 is given, then default shuffling is used: bit-wise
+  shuffling for buffers with item size 1, byte-wise shuffling otherwise.
+  Shuffling is turned off completely when the method value is 0.
+- The size of the compressed blocks is an integer number of bytes. When
+  it is set to 0, an automatic size will be used.
+- The name of the compression algorithm is a string identifier corresponding
+  to one of the algorithms supported by Blosc, e.g. "lz4", "zstd", "blosclz",
+  "zlib" or "snappy".
```

Comment:
Suggset moving this content into the section below on "Configuring codec in array metadata" as it provides a bit more explanation about those configuration parameters.